quinn-proto/src/crypto/ring.rs

```diff
@@ -4,43 +4,23 @@ use ring::{aead, hkdf, hmac};
 use crate::{
     config::ConfigError,
     crypto,
+    crypto::KeyPair,
     packet::{PacketNumber, LONG_HEADER_FORM},
     shared::ConnectionId,
     Side,
 };
 
 /// Keys for encrypting and decrypting packet payloads
-pub struct Crypto {
-    pub(crate) local_secret: hkdf::Prk,
-    local_iv: Iv,
-    sealing_key: aead::LessSafeKey,
-    pub(crate) remote_secret: hkdf::Prk,
-    remote_iv: Iv,
-    opening_key: aead::LessSafeKey,
+pub struct PacketKey {
```

Comment:
(Maybe rename to `PayloadKey`?)

Reply:
As above, I think the draft's use of "packet protection key" favors the current form.

Reply:
Yup.